branchpro/__init__.py

```diff
@@ -0,0 +1,12 @@
+"""branchpro is a Branching Processes modelling library.
+It contains functionality for modelling, simulating, and visualising the
+number of cases of infections by day during an outbreak of the influenza virus.
+"""
+
+# Import version info
+from .version_info import VERSION_INT, VERSION  # noqa
+
+# Import main classes
+from .models import ForwardModel, BranchProModel    # noqa
```

Comment:
I think these lines will fail until we get the next pull requests in. I would remove them for now, and add them back in within those pull requests.

Reply:
Noted; made the required change